src/Services/AppInstallActivationHandler.cs

```diff
@@ -28,20 +29,32 @@ public class AppInstallActivationHandler : ActivationHandler<ProtocolActivatedEv
     private readonly IWindowsPackageManager _windowsPackageManager;
     private readonly PackageProvider _packageProvider;
     private readonly SetupFlowOrchestrator _setupFlowOrchestrator;
+    private readonly WindowEx _mainWindow;
+    private readonly ISetupFlowStringResource _setupFlowStringResource;
     private static readonly char[] Separator = [','];
 
+    public enum ActivationQueryType
+    {
+        Search,
+        WingetURIs,
+    }
+
     public AppInstallActivationHandler(
         INavigationService navigationService,
         SetupFlowViewModel setupFlowViewModel,
         PackageProvider packageProvider,
         IWindowsPackageManager wpm,
-        SetupFlowOrchestrator setupFlowOrchestrator)
+        SetupFlowOrchestrator setupFlowOrchestrator,
+        ISetupFlowStringResource setupFlowStringResource,
+        WindowEx mainWindow)
```

Comment:
You're probably going to need to follow the changes here: https://github.com/microsoft/devhome/pull/2934